packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -933,6 +922,19 @@ export async function webxdcStartUpCleanup() {
   }
 }
 
+/**
+ * Make sure a default size doesn't extend the size
+ * of the primary display work area.
+ */
+function adjustBounds(size: Size): Size {
```

Comment:
@Simon-Laux So this doesn't work now, since it will omit x and y, if they were set in `lastBounds`. Hilfe bitte!

Reply:
I guess TypeScript doesn't complain on this since a `Bounds` is a `Size` but a `Size` isn't a `Bounds`. So what I'd like to do is to accept both a `Bounds` and a `Size` and return x and y as well, if they are present.

Reply:
for the main window it calculates the middle of the screen

Reply:
anyways I thin we could do sth like this?
```ts
const last_bounds:Bounds|null = getLastBounds(accountId,msgId)
const size:Size = adjustDefaultSize(lastBounds || defaultSize)
const bounds:Bounds = {...(last_bounds||{}), ...size}
```

Reply:
`Partial<T>` makes all properties optional

Reply:
> anyways for now we could do sth like this?

That looks pretty nice actually. But we don't do anything with x and y. Maybe we can postpone this?